### PR TITLE
shehadak/lm1b

### PR DESCRIPTION
**Pull request by shehadak**
Changes replicated from https://github.com/brain-score/language/pull/88. Also added type hinting following https://github.com/brain-score/language/pull/88#pullrequestreview-1154943685.

### CODE REVIEW

**Review by mschrimpf — APPROVED** [review]

File: brainscore_language/models/lm1b/test.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import numpy as np
+import pytest
+
+from brainscore_language import load_model
+from brainscore_language.artificial_subject import ArtificialSubject
+
+
+# @pytest.mark.memory_intense
```
Comment (mschrimpf) [inline brainscore_language/models/lm1b/test.py:8]: dead code

File: brainscore_language/models/lm1b/test.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+)
+def test_reading_times(model_identifier, expected_reading_times):
+    model = load_model(model_identifier)
+    text = ["the", "quick", "brown", "fox", "jumps", "over", "the", "lazy"]
+    model.start_behavioral_task(task=ArtificialSubject.Task.reading_times)
+    reading_times = model.digest_text(text)["behavior"]
+    np.testing.assert_allclose(reading_times, expected_reading_times, atol=0.0001)
+
+
+# @pytest.mark.memory_intense
```
Comment (mschrimpf) [inline brainscore_language/models/lm1b/test.py:35]: dead code

File: brainscore_language/models/lm1b/test.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+)
+def test_next_word(model_identifier, expected_next_words):
+    model = load_model(model_identifier)
+    text = ["the president", "arrived to the", "White", "House", "after a long", "day"]
+    model.start_behavioral_task(task=ArtificialSubject.Task.next_word)
+    next_word_predictions = model.digest_text(text)["behavior"]
+    np.testing.assert_array_equal(next_word_predictions, expected_next_words)
+
+
+# @pytest.mark.memory_intense
```
Comment (mschrimpf) [inline brainscore_language/models/lm1b/test.py:50]: dead code

**Comment by mschrimpf** [comment]
@shehadak I'm guessing this currently fails because the `@pytest.mark.memory_intense` markers have been commented out (see above review).

**Review by kvfairchild — APPROVED** [review]
(Approve assuming checks pass!)

**Pull request merged**